<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item alignment="Qt::AlignTop">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="LeftOperation">
       <item>
        <widget class="QGroupBox" name="gp_file_option">
         <property name="title">
          <string>文件选项</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="btn_openImg">
            <property name="text">
             <string>打开图片</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gp_ability_option">
         <property name="title">
          <string>功能选项</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QCheckBox" name="ck_cut">
            <property name="text">
             <string>图像压缩</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="ck_noline">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>去除杂线</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="ck_print">
            <property name="text">
             <string>印刷体</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="ck_join">
            <property name="text">
             <string>图像合并</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gp_threshold">
         <property name="title">
          <string>阈值管理</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="6" column="2">
           <widget class="QLabel" name="sb_ratio_value">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>0.30</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSlider" name="sb_ratio">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="sb_binary_value">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>120</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSlider" name="sb_binary">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>120</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="sb_binary_text">
            <property name="text">
             <string>二值阈值</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="sb_ratio_text">
            <property name="text">
             <string>字体斜率</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="RightOperation">
       <item row="0" column="1">
        <widget class="QGroupBox" name="gp_show_image">
         <property name="title">
          <string>图像预览区域</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lbl_image_origin">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_image_after">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextEdit" name="text_result">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>识别结果将在此处显示</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2" alignment="Qt::AlignTop">
        <widget class="QGroupBox" name="gp_operate_image">
         <property name="title">
          <string>图像操作</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="btn_split">
            <property name="text">
             <string>裁剪图片</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_gray">
            <property name="text">
             <string>灰度预览</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_brinary">
            <property name="text">
             <string>二值预览</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_location">
            <property name="text">
             <string>位置预览</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_recognize">
            <property name="text">
             <string>文字识别</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btn_openImg</tabstop>
  <tabstop>ck_noline</tabstop>
  <tabstop>ck_print</tabstop>
  <tabstop>ck_cut</tabstop>
  <tabstop>ck_join</tabstop>
  <tabstop>sb_binary</tabstop>
  <tabstop>sb_ratio</tabstop>
  <tabstop>btn_split</tabstop>
  <tabstop>btn_gray</tabstop>
  <tabstop>btn_brinary</tabstop>
  <tabstop>btn_location</tabstop>
  <tabstop>btn_recognize</tabstop>
  <tabstop>text_result</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
